.github/workflows/mautic-asset-upload/upload.php

```diff
@@ -76,7 +76,7 @@
     'storageLocation' => 'local',
     'file'            => $response['file']['name'],
     'category'        => $assetCategoryId,
-    'isPublished'     => true,
+    'isPublished'     => false,
```

Comment:
Should not be this part of this PR  I think

Reply:
@fedys I'm also confused what is it doing here. I can see it's coming from https://github.com/mautic/mautic/commit/d983a889681096a25f2674f6ed63689ddf337add. Do you remember what is the story here?

Reply:
@escopecz I think I was just fixing some PR that included unrelated changes.

Reply:
There are more changes not related to this PR for example:

https://github.com/mautic/mautic/pull/8794/files#diff-e192a95a44e25a558822230a82b49a099f01a1f2a6ffecf274b06b1579e7a94cR36
https://github.com/mautic/mautic/pull/8794/files#diff-fb63f85c5c793dbfedafce713ba68125484bd35d7cab18c3e0493a359a26cb8bL222